### PR TITLE
chore: visual refresh for picker component

### PR DESCRIPTION
**Pull request by khamudom**
# Pull Request

## 📖 Description

Before (rest, hover, focused)
image
After
image

Before screenshot on the picked items
image
After
image

Before screenshot on the picker with picked items
image
After
image

### General

- [ ] I have included a change request file using `$ yarn change`
- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

### CODE REVIEW

File: packages/web-components/fast-components/docs/api-report.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ import { NumberFieldOptions } from '@microsoft/fast-foundation';
 import { Picker } from '@microsoft/fast-foundation';
 import { PickerList } from '@microsoft/fast-foundation';
 import { PickerListItem } from '@microsoft/fast-foundation';
-import { PickerMenu } from '@microsoft/fast-foundation';
+import { PickerMenu as PickerMenu_2 } from '@microsoft/fast-foundation';
```
Comment (scomea) [inline packages/web-components/fast-components/docs/api-report.md:60]: "PickerMenu_2" ?

Comment (bheston) [reply packages/web-components/fast-components/docs/api-report.md:60]: Setting the fill color of a flyout requires creating a subclass of the foundation component. This is how the API report represents both the foundation and the specific components.

File: packages/web-components/fast-components/src/picker/index.ts [hunk 2]
```diff
@@ -1,21 +1,37 @@
...
+import { fillColor, neutralLayerFloating } from "../design-tokens";
 import { pickerStyles } from "./picker.styles";
 import { pickerMenuStyles } from "./picker-menu.styles";
 import { pickerMenuOptionStyles } from "./picker-menu-option.styles";
 import { pickerListStyles } from "./picker-list.styles";
 import { pickerListItemStyles } from "./picker-list-item.styles";
 
+/**
+ * The FAST picker menu class
+ * @internal
```
Comment (bheston) [inline packages/web-components/fast-components/src/picker/index.ts:22]:
```suggestion
 * @public
```

File: packages/web-components/fast-components/src/picker/picker-list-item.styles.ts [hunk 3]
```diff
@@ -6,16 +6,22 @@ import {
     FoundationElementDefinition,
 } from "@microsoft/fast-foundation";
 import { SystemColors } from "@microsoft/fast-web-utilities";
+import {} from "..";
```
Comment (bheston) [inline packages/web-components/fast-components/src/picker/picker-list-item.styles.ts:9]: Remove
```suggestion
```

File: packages/web-components/fast-components/src/picker/picker-menu-option.styles.ts [hunk 4]
```diff
@@ -1,21 +1,20 @@
 import { css, ElementStyles } from "@microsoft/fast-element";
 import {
     ElementDefinitionContext,
-    focusVisible,
     forcedColorsStylesheetBehavior,
     FoundationElementDefinition,
 } from "@microsoft/fast-foundation";
 import { SystemColors } from "@microsoft/fast-web-utilities";
+import { neutralFillStealthFocus, neutralForegroundFocus, strokeWidth } from "..";
```
Comment (bheston) [inline packages/web-components/fast-components/src/picker/picker-menu-option.styles.ts:8]: Fix the import path. Should be a file path, not just ..

File: packages/web-components/fast-components/src/picker/picker-menu-option.styles.ts [hunk 5]
```diff
@@ -35,49 +34,49 @@ export const pickerMenuOptionStyles: (
...
-        color: ${foregroundOnAccentActive};
-    }
+        :host(:active) {
+            background: ${neutralFillStealthActive};
+            color: ${neutralForegroundActive};
+        }
 
+        :host(:not([aria-selected="true"]):hover),
+        :host([aria-selected="true"]) {
+            background: ${neutralFillStealthFocus};
```
Comment (bheston) [inline packages/web-components/fast-components/src/picker/picker-menu-option.styles.ts:70]: I think we were using plain neutralFill for selected states,
```suggestion
            background: ${neutralFillRest};
```

**Review by bheston — APPROVED** [review]

File: packages/web-components/fast-components/src/picker/picker-menu-option.styles.ts (hunk 5 above)
Comment (khamudom) [reply packages/web-components/fast-components/src/picker/picker-menu-option.styles.ts]: updated

File: packages/web-components/fast-components/src/picker/index.ts (hunk 2 above)
Comment (khamudom) [reply packages/web-components/fast-components/src/picker/index.ts]: updated

**Review by scomea — APPROVED** [review]

**Pull request merged**